src/tests/ci-configs/vision-segmentation-simple.yaml

```
@@ -0,0 +1,15 @@
+seed: 666
+data_path: ~/data
+train_batch_size: 2
+eval_batch_size: 2
+num_workers: 2
+max_epochs: 2
+train_epoch_length: 2
+eval_epoch_length: 2
```

```suggestion
train_epoch_length: 4
eval_epoch_length: 4
```
as original cmd line options